pytest-sugar is not needed to run tests.
python-memcached is optional test dependendency and we do not have it packaged
yet.  See also https://github.com/cherrypy/cherrypy/pull/1984

--- CherryPy-18.9.0/setup.py.orig
+++ CherryPy-18.9.0/setup.py
@@ -94,7 +94,6 @@
             'pytest>=5.3.5',
             'pytest-cov',
             'pytest-forked',
-            'pytest-sugar',
             'path.py',
             'requests_toolbelt',
             'pytest-services>=2',
--- CherryPy-18.9.0/tox.ini.orig
+++ CherryPy-18.9.0/tox.ini
@@ -21,7 +21,6 @@
 extras =
     testing
     routes_dispatcher
-    memcached_session
 whitelist_externals = mkdir
 
 [python-cli-options]
